src/index.js

```diff
@@ -1,5 +1,3 @@
-/* eslint-disable no-unused-vars */
-/* eslint-disab */
-
 import './style.css';
-import Task from './modules/add.js';
+// eslint-disable-next-line no-unused-vars, quotes
```

Comment:
- [ ] [OPTIONAL] Kindly remove the `eslint-disable` and try to fix the error manually because this defies the benefit of using linters.